src/lit-html.ts

```diff
@@ -624,3 +618,19 @@ export class TemplateInstance {
     return fragment;
   }
 }
+
+/**
+ * Removes nodes, starting from `startNode` (inclusive) to `endNode`
+ * (exclusive), from `container`.
+ */
+export const removeNodes =
```

Comment:
Similar to my comment in the SVG PR: https://github.com/PolymerLabs/lit-html/pull/164#discussion_r148941728

I'd change the signature to:

```ts
const removeNodes = (container: Node, start: Node, end: Node | null = null): void => {
```

Reply:
Done.